tests/test_setpoint_constraints.py

```diff
@@ -65,11 +65,6 @@ def test_run_small_ates_timed_setpoints_2_changes(self):
         Checks:
         - That the setpoint does change twice over window length if 2 is specified
 
-        Missing:
-        - This test runs extremely slow in current master, should be fixed!
-        _ this test does not check for funtioning with varying timestep sizes, this or another test
-        should be adapted to do so or a seperate test for that should be created.
```

Comment:
I don't see an adaptation for this or a new test

Reply:
This was actually already in, the timeline had varying timestep size